lib/tasks/bulwark/repo.rake

```diff
@@ -0,0 +1,77 @@
+namespace :bulwark do
+  namespace :repo do
+    desc 'Purges Repo. Removes db object, git repository and special remotes.'
+    task purge: :environment do
+      abort(Rainbow('Incorrect arguments. Pass arklist=/path/to/file or arks=seperated,by,commas').red) if ENV['arklist'].blank? && ENV['arks'].blank?
+
+      # Pass in ARK(s) for digital object(s) to be deleted.
+      arks = open(ENV['arklist']).map(&:strip!) if ENV['arklist'].present?
```

Comment:
catching an exception here from `open()` (e.g., `FileNotFound`) would be helpful here if, say, the user mistypes a path

Reply:
Will do